app/controllers/volunteer_applications_controller.rb

```diff
@@ -0,0 +1,49 @@
+class VolunteerApplicationsController < ApplicationController
+  def new
+    @volunteer_application = VolunteerApplication.new
+  end
+
+  def create
+    @volunteer_application = VolunteerApplication.new
+                             (volunteer_application_params)
+    if @volunteer_application.save
+      flash.now[:success] =
```

Comment:
This doesn't need to use `.now` because you redirect. The one below does because you call `render`